src/optimizer/join_order/relation_statistics_helper.cpp

```diff
@@ -241,7 +242,25 @@ RelationStats RelationStatisticsHelper::CombineStatsOfNonReorderableOperator(Log
 		default:
 			break;
 		}
+		break;
+	}
+	case LogicalOperatorType::LOGICAL_UNION:
+	case LogicalOperatorType::LOGICAL_INTERSECT:
+	case LogicalOperatorType::LOGICAL_EXCEPT: {
+		auto &setop = op.Cast<LogicalSetOperation>();
+		if (setop.type == LogicalOperatorType::LOGICAL_EXCEPT) {
```

Comment:
Bit of a nitpick but it seems like this is essentially switching twice on the same type - can we simplify this?